site/components/map/ServicesMap.tsx

```diff
@@ -288,7 +290,9 @@ const ServicesMap = ({
                                 ? { stopTypes: ["FER", "FBT"] }
                                 : vehicleMode === Modes.rail
                                   ? { stopTypes: ["RLY"] }
-                                  : { stopTypes: ["undefined"] }),
+                                  : vehicleMode === VehicleMode.coach
```

Comment:
Can we use the new `getStopTypesByVehicleMode` function here as well?

Reply:
done